selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceSettingManagement.java

```diff
@@ -181,7 +185,44 @@ public void declineRequest(String user) {
     info("Verify that the member is shown in member list");
     evt.waitForElementNotPresent(ELEMENT_SPACE_DELETE_USER_BTN.replace("${user}", user), 2000, 1);
   }
-
+  /*** Search User in People
+   *
+   */
+    public void searchUsersPeople(String user) {
+        info("--Search user To connect" + user + "--");
+            ($(byXpath("(//input[@placeholder=\"Name\"])[2]"))).setValue(user);
+            ($(byXpath("//button[text()='Search']"))).waitUntil(Condition.visible, Configuration.timeout).click();
+            ($(byXpath("//div[@class=\"connectionBtn clearfix\"]/button"))).waitUntil(Condition.visible, Configuration.timeout).click();
+            ($(byXpath("//button[text()=\"Cancel Request\"]"))).shouldNot(Condition.visible);
+    }
+    /*** Connect Searched User
+     *
+     */
+    public void connectSearchedUser() {
+             if ($(byXpath("//button[text()=\"Cancel Request\"]")).waitUntil(Condition.visible,Configuration.timeout).exists()) {
+                 $(byXpath("//button[text()=\"Cancel Request\"]")).click();
+               $(byXpath("//button[text()=\"Cancel Request\"]")).shouldNot(Condition.visible);
+             ($(byXpath("//div[@class=\"connectionBtn clearfix\"]/button"))).waitUntil(Condition.visible, 7000).click();
+               $(byXpath("//button[text()=\"Cancel Request\"]")).should(Condition.visible);
+             }
+             else {
+                 ($(byXpath("//div[@class=\"connectionBtn clearfix\"]/button"))).waitUntil(Condition.visible, 7000).click();
+                 $(byXpath("//button[text()=\"Cancel Request\"]")).should(Condition.visible);
+             }
```

Comment:
Indentation is wrong.

Reply:
I will delete this function because it is unused, done.